### ISSUE TITLE
Set up resource usage monitoring

### ISSUE THREAD

**Issue by phargogh** [natcap/data.naturalcapitalproject.stanford.edu#16]
It is very possible that we will one day exhaust the capabilities of this VM, and it is very likely that some component of this service will need to be scaled out.  Resource usage monitoring will enable us to more clearly identify which service(s) are bottlenecking the rest of the application.

**Comment by phargogh** [comment]
I initiated the Ops Agent installation process on the VM, which should allow us to view individual process usage on the VM.

**Comment by phargogh** [comment]
The Ops Agent installation is taking _way_ longer than I expected.  If it hasn't finished installing by tomorrow, it might be worth restarting the VM ... but the catch there is then that I'll need to finish uninstalling all of the other CKAN stuff from v1 of CKAN.

**Comment by phargogh** [comment]
Ops agent still hadn't installed. I just uninstalled a bunch of stuff no longer needed and updated all system packages which included updating some google packages, including the ops agent.  Hopefully that'll set us up with working monitoring.

**Comment by phargogh** [comment]
Looks like the missing step was that I needed to enable the "Stackdriver Monitoring API".  I guess I was required to create an application data service account, skipping the "Grant this service account access to project" and "Grant users access to this service account" sections.  I created the service account but didn't add the service account to the CKAN VM, yet I can clearly see that data is flowing to the monitoring dashboard.

So I think we're done!
